peps/pep-0730.rst

```diff
@@ -0,0 +1,309 @@
+in the following values:
+
+* ``sysname`` - ``Darwin``
+
+* ``release`` - The Darwin kernel version (e.g., ``"22.6.0"``)
+
+``sysconfig``
+'''''''''''''
+
+The ``sysconfig`` module will use the minimum iOS version as part of ``sysconfig.get_platform()`` identifier (e.g., ``iOS-12.0-iphoneos-arm64``). The ``sysconfigdata_name`` and Config makefile will follow the same patterns as existing platforms (using ``sys.platform``, ``sys.implementation._multiarch`` etc) to construct identifiers.
+
+Subprocess support
+------------------
+
+iOS will leverage the pattern for disabling subprocesses established by WASI/Emscripten. The ``subprocess`` module will raise an exception if an attempt is made to start a subprocess ``os.fork`` and ``os.spawn`` calls will raise an ``OSError``.
+
+Dynamic module loading
+----------------------
+
+To accommodate iOS dynamic loading, the ``importlib`` bootstrap will be extended to add a metapath finder that can convert a request for a Python binary module into a Framework location. This finder will only be installed if ``sys.platform == "ios"``.
+
+This finder will convert a Python module name (e.g., ``foo.bar._whiz``) into a unique Framework name by replacing the dots with underscores (i.e., ``foo_bar__whiz.framework``). A framework is a directory; the finder will look for ``_whiz.dylib`` in that directory.
+
+CI resources
+------------
+
+Github Actions is able to host iOS simulators on their macOS machines, and the iOS simulator can be controlled by scripting environments. The free tier currently only provides x86_64 macOS machines; however ARM64 runners have recently become available on paid plans [5]_.
+
+If Github Actions resources are insufficient or not viable for cost reasons, Anaconda has offered to provide resources to support CI requirements.
+
+Packaging
+---------
+
+iOS will not provide a "universal" wheel format. Instead, wheels will be provided on a for each ABI-arch combination. At present, no binary merging is required. There is only 1 on-device architecture; and simulator binaries are not considered to be distributable artefacts, so only 1 architecture is needed to build a simulator.
+
+iOS wheels will use tags:
+
+* ``iOS_12_0_iphoneos_arm64``
+* ``iOS_12_0_iphonesimulator_arm64``
+* ``iOS_12_0_iphonesimulator_x86_64``
+
+In these tags, "12.0" is the minimum supported iOS version. The choice of minimum supported iOS version is a decision of whoever compiles CPython for iOS. At time of writing, iOS 12.0 exposes most significant iOS features, while reaching near 100% of devices.
+
+These wheels can include binaries modules in-situ (i.e., co-located with the Python source, in the same way as wheels for a desktop platform); however, they will need to be post-processed as part of inin the samewill need to be post-processed into the "Frameworks" location required for distribution. This can be automated with an Xcode build step.
```

Comment:
```suggestion
These wheels can include binary modules in-situ (i.e., co-located with the Python source, in the same way as wheels for a desktop platform); however, they will need to be post-processed as binary modules need to be moved into the "Frameworks" location for distribution. This can be automated with an Xcode build step.
```
Not sure about the second sentence but the existing wording was broken